### PR TITLE
Readme update

### LINKED ISSUE #25: Put 'clone and go' guide in the readme
Should have step by step instructions to cloning and getting started for mac and pc environments. 

In other words: It's my first day at code for dc, I've got the readme open, what do I need to install and set up to be able to run a local server and start wrenching around?

### PR DESCRIPTION
**Pull request by colinxfleming**
Adds a badly needed readme update, with the following:
* Local setup info
* Draft PR / contribution policy
* Updated project description
* Depreciates the jekyll stuff since we're on rails now

Still needs some clarity on what we need from nontechnical people (testing, designers, etc) but idk if we're quite there yet. Something we should figure out soon though, since I think it'll start losing us potential person-power soon.

@mebates could you spot check the How Do I Contribute and project descriptions? 

@lwaldsc if you've got time, could you try out the Setting Things Up Locally portion and make sure the Cloud9 stuff works for you? 

@mattbailey0 would welcome any feedback you have on this, as well. 

Fixes #25, #10, #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,48 @@
...
-* When you have made changes and you want to upload them onto Github, add and commit your changes by using the following commands: `git add` and `git commit`
-* Push to origin master by inputting `git push -u origin master`
-* Once you have pushed your changes, make a pull request.
+We're generally looking for people comfortable or interested in the following:
+* Ruby on Rails (a good starter kit: [CodeAcademy's course](http://www.codecademy.com/learn/learn-rails))
+* JQuery / HTML / CSS
+* NoSQL / flat data modeling
+* Nonprofit data issues 
+
+Get started with the how-and-why of the project by [checking out DCAF](http://dcabortionfund.org), checking out [DCAF Case Manager Lisa's explanation of DCAF's business logic](TK), looking at the design team's [InVision prototype](https://projects.invisionapp.com/share/6757W6WFJ), and reading the `#dcaf_case_management` channel on Slack. Hit up @colinxfleming and Mollie with any questions.
```
Comment (charleshuang80) [inline README.md]: what is TK? Is this missing a link?

Comment (charleshuang80) [inline README.md]: Is this @Mollie, or does she have another name on the channel?

File: README.md [hunk 2]
```diff
@@ -3,18 +3,48 @@
...
+
+    Sign into `https://c9.io/` and create a new workspace
+    Clone from `git@github.com:{your_github_username}/dcaf_case_management.git` and select the Rails option
+    Once created, `$ bundle install` from the terminal
+    `$ mongod` to start MongoDB
+    Hit the `Run Project` button up top
+
+If you do prefer a local environment, do the following: 
+
+    # Install Ruby, Rails and MongoDB
```
Comment (charleshuang80) [inline README.md]: Might want to indicate that setting up MongoDB is described below.

File: README.md [hunk 3]
```diff
@@ -22,12 +52,14 @@ Current wireframe assets are available here:
 
 ## Setting up MongoDB
 
-* Install MongoDB locally if you haven't (`$ brew install mongodb`, for example)
+* Install MongoDB locally if you haven't (`$ brew install mongodb`, for example, or the [linux instructions](https://docs.mongodb.org/manual/tutorial/install-mongodb-on-ubuntu/))
 * `$ bundle install` to install necessary gems
 * Create a folder in the root directory for the MongoDB database. `$ sudo mkdir -p /data/db`
 * Allow for MongoDB read/write permissions `$ sudo chmod 777 data/db`
```
Comment (charleshuang80) [inline README.md]: requires a '/' in front of the 'data/db', just as in the line above.

File: README.md (hunk 1 above)
Comment (colinxfleming) [inline README.md]: Yeah, it's the Mollie from Slack. Wanted to ask her before I linked her in the readme.

The TK is a doc we're ironing out still to provide a 101-level overview of DCAF. I'm throwing the draft version in the wiki shortly, thank you for this reminder to update the link!

File: README.md (hunk 2 above)
Comment (colinxfleming) [inline README.md]: You are right about this -- excellent call.

File: README.md (hunk 3 above)
Comment (colinxfleming) [inline README.md]: Excellent catch! Fixed.

**Pull request merged**